gcp/README.md

```diff
@@ -673,3 +673,55 @@ resolved/update is done.
 If this downtime will be disruptive to your work ("We're giving a demo for the
 Prime Minister at that time!"), please let me know immediately.
 ```
+
+#### External backups
```

Comment:
Why is this a subsection of Downtime Procedures?

Reply:
Wrong nesting, I have decrease it: https://github.com/gpii-ops/gpii-infra/pull/432/commits/38542c9230965b1c36279bdf603dbb40192b1a3e